autoload/test/javascript/lab.vim

```diff
@@ -0,0 +1,50 @@
+if !exists('g:test#javascript#lab#file_pattern')
+  let g:test#javascript#lab#file_pattern = '\vtest/.*\.js$'
+endif
+
+function! test#javascript#lab#test_file(file) abort
+  return a:file =~# g:test#javascript#lab#file_pattern
+endfunction
+
+function! test#javascript#lab#test_file(file) abort
+  if a:file =~# g:test#javascript#lab#file_pattern
+    if exists('g:test#javascript#runner')
+      return g:test#javascript#runner == 'lab'
+    else
+      return executable('lab')
```

Comment:
Exactly, this is what I did for my PR https://github.com/janko-m/vim-test/pull/178 where the test file format was the same but the runner is determined by which one is installed in the project binary directory